packages/terra-action-header/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Updated `--terra-action-header-icon-top` variable, added `--terra-action-header-icon-ms-top` variable to only apply top styling in Microsoft browsers.
```

Comment:
This should be updated to "**Removed** `--terra-dialog-header-close-icon-margin-top` ..." as per https://github.com/cerner/terra-core/pull/2523#discussion_r321389206

Reply:
`--terra-action-header-icon-top` variable was not removed just value was updated as per this [comment](https://github.com/cerner/terra-core/pull/2523/files#r325135405)